flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/table/MySqlDeserializationConverterFactory.java

```diff
@@ -148,4 +146,8 @@ public Object convert(Object dbzObj, Schema schema) throws Exception {
             return Optional.empty();
         }
     }
+
+    private static boolean hasFamily(LogicalType logicalType, LogicalTypeFamily family) {
```

Comment:
This is replaced by `arrayType.getElementType().is(LogicalTypeFamily.CHARACTER_STRING)`.

Reply:
Same as above